nUpdate.ProvideTAP/Updating/UpdateManager.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright ?Dominic Beger 2017
```

Comment:
Has this happened when downloading the file? The original code seems to be fine.
In any case, thank you for correcting this again.

I'd appreciate, if your commit messages would fit to the others. That means, that no past participle should be used, so that the commit message is compatible with the sentence "If applied, this commit will ... ".
This would now result in "If applied, this commit will fixed copyright symbol" which is a bit strange, of course. So "Fix broken copyright symbol" should be fine.